apps/gazettes-aleph/gazettes-aleph.yml

```diff
@@ -0,0 +1,105 @@
+---
+- hosts:
+    - gazettes_aleph_web
+  become: yes
+  roles:
+    - dokku_bot.ansible_dokku
+
+  vars:
+    app_name: "gazettes-aleph"
+    dokku_daemon_install: false # We install this using a dedicated playbook
+    dokku_plugins:
+      - name: letsencrypt
+        url: https://github.com/dokku/dokku-letsencrypt.git
+      - name: elasticsearch
+        url: https://github.com/dokku/dokku-elasticsearch.git
+    postgres_hostname: postgresql11-prod.cnc362bhpvfe.eu-west-1.rds.amazonaws.com
+    postgres_database: aleph
+    postgres_user: "{{ lookup('passwordstore', 'apps/aleph/gazettes-aleph/{{ env_name }}/POSTGRES subkey=username')}}"
+    postgres_password: "{{ lookup('passwordstore', 'apps/aleph/gazettes-aleph/{{ env_name }}/POSTGRES')}}"
+
+  tasks:
+    - name: "Dokku app exists"
+      dokku_app:
+        app: "{{ app_name }}"
+
+    - name: "set vm.max_map_count to 262144 in sysctl"
+      sysctl: "name={{ item.key }} value={{ item.value }}"
+      with_items:
+        - key: "vm.max_map_count"
+          value: "262144"
+
+    # TODO: Get the following vars exported properly so that the correct image is used
+    # TODO: The following shell command is not idempotent
+    - name: "Dokku elasticsearch"
+      vars:
+        ELASTICSEARCH_IMAGE: "openup/elasticsearch-0.90"
+        ELASTICSEARCH_IMAGE_VERSION: "latest"
+      shell: "dokku elasticsearch:create openup-es"  # Is there a way to do this using dokku plugin?
```

Comment:
dokku plugin can't do this yet, so I think it's best if we do this by hand for now - let's document the manual steps in a readme file, e.g.

apps/gazettes-aleph/README.md
apps/gazettes-aleph/playbook.yml

Reply:
my thinking here is ansible should represent the authoritative concrete way that the app is deployed right now, while the app repo can continue representing the generic way that an app can be deployed and configured regardless of environment